.travis.yml

```diff
@@ -252,6 +252,25 @@ matrix:
     env:
       - FEDORA=23
       - KITCHEN_YAML=.kitchen.travis.yml
+  - rvm: 2.3.1
+    services: docker
+    sudo: required
+    gemfile: kitchen-tests/Gemfile
+    before_install:
+      - gem update --system $(grep rubygems omnibus_overrides.rb | cut -d'"' -f2)
+      - gem install bundler -v $(grep bundler omnibus_overrides.rb | cut -d'"' -f2)
+    bundler_args: --without changelog development docgen guard integration maintenance omnibus_package tools aix bsd mac_os_x solaris windows --frozen
+    before_script:
+      - sudo iptables -L DOCKER || ( echo "DOCKER iptables chain missing" ; sudo iptables -N DOCKER )
+      - cd kitchen-tests
+      - git clone https://github.com/learn-chef/awesome_customers_ubuntu cookbooks/awesome_customers_ubuntu
+    script:
+      - bundle exec kitchen test fedora-23
```

Comment:
yeah i'm not even doing anything with fedora this PR that was copypasta that has been modified and is gone now